node-tests/blueprints/helper-addon-test.js

```diff
@@ -20,4 +20,14 @@ describe('Acceptance: ember generate and destroy helper-addon', function() {
           .to.contain("export { default, fooBar } from 'my-addon/helpers/foo-bar';");
       }));
   });
+  
+  it('in-addon helper-addon foo-bar --pod', function() {
+    var args = ['helper-addon', 'foo-bar', '--pod'];
+
+    return emberNew({ target: 'addon' })
+      .then(() => emberGenerateDestroy(args, _file => {
+        expect(_file('app/helpers/foo-bar.js'))
+          .to.contain("export { default, fooBar } from 'my-addon/helpers/foo-bar';");
+      }));
+  });
```

Comment:
it's certainly useful to have these tests here. I'm just not sure what the expected path layout for Pods in these cases is.

/cc @rwjblue @dgeb